### PR TITLE
[app-test] fix invalid conf file

### PR DESCRIPTION
**Pull request by jaeyun-jung**
set valid conf file path for nnstreamer

**Changelogs**
* Version 2:
  - remove unnecessary env variables to get sub-plugins

* Version 1:
  - fix invalid conf path

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

File: ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh [hunk 1]
```diff
@@ -142,7 +142,7 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
     echo -e "[DEBUG] GST_PLUGIN_PATH is '$GST_PLUGIN_PATH'"
 
     # nnstreamer env variables and include paths
-    export NNSTREAMER_CONF=$NNST_ROOT
+    export NNSTREAMER_CONF=$NNST_ROOT/nnstreamer.ini
```
Comment (leemgs) [inline ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: What is a goal of this file (e.g., ..../nnstreamer.ini)?

Comment (jaeyun-jung) [reply ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: .ini file includes the path for sub-plugins. I updated this commit to remove unnecessary env values about sub-plugins path.

Comment (leemgs) [reply ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: It seems that the updated contents is better. Does it mean that  sub-plugins will be maintained by nnstreamer.ini file (without  environment variables such as the NNSTREAMER_****)?

Comment (jaeyun-jung) [reply ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: yes, installed ini file includes each sub-plugin paths.
```
in meson build script
subplugin_install_prefix = join_paths(nnstreamer_prefix, 'lib', 'nnstreamer')
nnstreamer_conf.set('SUBPLUGIN_INSTALL_PREFIX', subplugin_install_prefix)
```
```
in nnstreamer.ini.in
[filter]
filters=@SUBPLUGIN_INSTALL_PREFIX@/filters/
customfilters=@SUBPLUGIN_INSTALL_PREFIX@/customfilters/

[decoder]
decoders=@SUBPLUGIN_INSTALL_PREFIX@/decoders/
```

Comment (leemgs) [reply ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: Acked-by : Geunsik Lim.

**Pull request merged**